org-eclipse-che-jdt-ls-extension/jdt.ls.extension.core/src/main/java/org/eclipse/che/jdt/ls/extension/core/internal/externallibrary/LibraryNavigation.java

```diff
@@ -121,8 +128,39 @@
     if (javaProject == null) {
       throw new IllegalArgumentException(format("Project for '%s' not found", projectUri));
     }
+
+    IMavenProjectRegistry mavenProjectRegistry = MavenPlugin.getMavenProjectRegistry();
```

Comment:
I don't really understand why we would need any maven-specific stuff, here. Can you explain, please?

Reply:
There are no external libraries for for maven projects with pom packaging.
That's why we have to recognize that and return default libs to sustain current che behavior.

Reply:
<sigh> fair enough.